### ISSUE TITLE
Writing contour plots to SVG takes 20 odd seconds

### ISSUE THREAD

**Issue by rajuah** [plotly/orca#231]
I am trying to use plotly (offline mode) in python along with orca to export contour plots to SVG. 
Given below is the function I use to create contour plots and export it to SVG
```
def plot_contour_filled_to_svg():
            fig = go.Figure(layout=layout)
            fig.add_contour(
                x=x_axis,
                y=y_axis,
                z=scan_data,
                contours=dict(showlines=False),
                colorscale="Jet",
            )
            pio.write_image(
                fig,
                "C:/Users/araju/PycharmProjects/contour_filled.svg",
            )
```
It takes roughly 20s to create an SVG every single time. Is this expected behavior?

I though maybe file IO part of the write command is causing that delay so instead of the `write_image`, I used the `to_image` to get a bytes representation of the same. That didnt help with the runtime either

I ran the orca server manually in debug mode (looked at a few posts in forums) but I dont know how to make sense of it.

![image](https://user-images.githubusercontent.com/17533956/57411923-d5145c00-71a3-11e9-8bc1-fb9314143b5c.png)

Can someone help me out?

**Comment by etpinard** [comment]
Thanks for writing in. Can you share the x,y,z data you're trying to export to help us debug?

**Comment by rajuah** [comment]
Thanks for the quick reply. I did some further digging into my issue. I work behind an enterprise firewall and I suspected that gating/slowing down the orca server somehow. So I tested the same script with same data on a Mac and Win PC outside of the enterprise network and was able to always get plotly+orca to export static images in under 0.3sec. 
So at this point I think orca + plotly are not to blame. They are working as they should. Thanks for the great work with orca and plotly 👍 :)
